### DISCUSSION TITLE
What is the difference between having demux and not having demux?

### DISCUSSION THREAD

**Discussion by KEIEK10** [PyAV-Org/PyAV#846] [1. Help]
Hi, I'm Japanese. I've started to study video analysis, but I'm having trouble understanding it. Please help me.
### What is the difference between having demux and not having demux?
MotionVectors will be displayed normally.
![Screenshot from 2021-11-29 07-33-13](https://user-images.githubusercontent.com/94222446/143788888-459238ba-e58e-4728-b207-9e3c30d97532.png)
Also, please explain how the looping process in the following image works.
![Screenshot from 2021-11-29 07-33-56](https://user-images.githubusercontent.com/94222446/143788891-afeeef4a-4cad-4097-a6fb-7c9855766408.png)

**Comment by sage-47** [comment]
in my opinion,  you can control the rate of decode frame by use demux packet, which can reduce cpu ranking.
